ecosystem/indexer/src/indexer/errors.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Error;
+
+type ErrorWithVersionAndName = (Error, u64, &'static str);
+
+#[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
nit: you should be able to delete this now, right? Everything appears to be used.